AnkiDroid/src/main/java/com/ichi2/themes/StyledProgressDialog.java

```diff
@@ -60,6 +60,10 @@ public static MaterialDialog show(Context context, CharSequence title, CharSeque
 
     public static MaterialDialog show(Context context, CharSequence title, CharSequence message,
             boolean cancelable, DialogInterface.OnCancelListener cancelListener) {
+        if (title.equals("")) {
```

Comment:
If title is null, then this fails

```suggestion
        if ("".equals(title)) {
```

Reply:
fixed